CHANGELOG.md

```diff
@@ -4,12 +4,18 @@ All notable changes to this project will be documented in this file. See [standa
 
 ## [Unreleased](https://github.com/dotenv-org/python-dotenv-vault/compare/v0.5.1...master)
 
-## 0.6.1
+## 0.6.2
 
 ### Changed
 
 - Fix fallback issue with gunicorn not respecting the current working directory when attempting to call `find_dotenv`. [#17](https://github.com/dotenv-org/python-dotenv-vault/pull/17)
 
+## 0.6.1
+
+### Changed
+
+- Look for .env.vault file at same location as .env file. Finds .env file anywhere in app (just like original python lib) [#13](https://github.com/dotenv-org/python-dotenv-vault/pull/13)
```

Comment:
One more nitpick. This comment should be under `0.6.1` and the one above should be under `0.6.2`. Then this PR should be ready to go.

Reply:
okay, i switched this around @motdotla . It should be in correct place now